package.json

```diff
@@ -34,11 +34,10 @@
     },
     "dependencies": {
         "pxt-common-packages": "8.4.9",
-        "pxt-core": "6.8.30",
-        "pxt-arcade-sim": "microsoft/pxt-arcade-sim.git#v0.8.1"
```

Comment:
This looks unnecessary, and appears to be rather new? Maybe just an errant npm i?